src/modules/sdbus/sdbus.c

```diff
@@ -16,6 +16,7 @@
 #endif
 #include <flux/core.h>
 #include <systemd/sd-bus.h>
+#include <fnmatch.h>
```

Comment:
trivial nit: extra space after "Problem:" in commit message

Reply:
Fixed, thanks.  I'll go ahead and set MWP.